static/js/containers/LearnerSearchPage.js

```diff
@@ -51,23 +57,31 @@ class LearnerSearchPage extends React.Component {
   };
 
   closeEmailComposeAndSend: Function = () => {
-    const { dispatch, email } = this.props;
-    console.log(email); // eslint-disable-line no-console
-    dispatch(clearEmailEdit());
-    dispatch(setEmailDialogVisibility(false));
+    const { dispatch, email: { email } } = this.props;
+    let errors = emailValidation(email);
+    dispatch(updateEmailValidation(errors));
+    if ( R.isEmpty(errors) ) {
+      console.log(email); // eslint-disable-line no-console
```

Comment:
yeah exactly, I merged it like this on Wednesday because I wanted to move forward on the UI work (and it's not so dependent on having the API in place anyway).